packages/sui-pde/src/components/feature.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types'
+import useFeature from '../hooks/useFeature'
+
+export default function Feature({
+  children,
+  featureKey,
+  attributes,
+  queryString
+}) {
+  const value = useFeature(featureKey, attributes, queryString)
+  return children(value)
```

Comment:
same as above... I think it helps to know what `value` actually is, atm...

```suggestion
  const isActive = useFeature(featureKey, attributes, queryString)
  return children({isActive})
```